src/SDK/Common/Retry/ExponentialWithJitterRetryPolicy.php

```diff
@@ -0,0 +1,154 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\SDK\Common\Retry;
+
+use InvalidArgumentException;
+use OpenTelemetry\SDK\Trace\SpanExporterInterface;
+
+final class ExponentialWithJitterRetryPolicy implements RetryPolicyInterface
+{
+    public const DEFAULT_MAX_ATTEMPTS = 5;
+    public const DEFAULT_INITIAL_BACKOFF = 1;
+    public const DEFAULT_MAX_BACKOFF = 10;
+    public const DEFAULT_BACKOFF_MULTIPLIER = 1.5;
+    public const DEFAULT_JITTER = 0.1;
+
+    private $maxAttempts;
+    private $initialBackoff;
+    private $maxBackoff;
+    private $backoffMultiplier;
+    private $jitter;
+    public $retryableStatusCodes;
+
+    public function __construct(
+        int $maxAttempts = self::DEFAULT_MAX_ATTEMPTS,
+        float $initialBackoff = self::DEFAULT_INITIAL_BACKOFF,
+        int $maxBackoff = self::DEFAULT_MAX_BACKOFF,
+        float $backoffMultiplier = self::DEFAULT_BACKOFF_MULTIPLIER,
+        float $jitter = self::DEFAULT_JITTER
+    ) {
+        $this->setMaxAttempts($maxAttempts);
+        $this->setInitialBackoff($initialBackoff);
+        $this->setMaxBackoff($maxBackoff);
+        $this->setBackoffMultipler($backoffMultiplier);
+        $this->setJitter($jitter);
+    }
+
+    public function shouldRetry(
+        int $attempt,
+        int $status
+    ): bool {
+        return $attempt < $this->maxAttempts &&
+            $status == SpanExporterInterface::STATUS_FAILED_RETRYABLE;
+    }
+
+    public function getDelay(int $attempt): int
+    {
+        // Dont wait for the first attempt
+        if ($attempt == 0) {
+            return 0;
+        }
+        // Initial exponential backoff in mili seconds
+        $delay = ($this->backoffMultiplier ** $attempt) * $this->initialBackoff * 1000;
+
+        // Adding jitter to exponential backoff
+        if ($this->jitter > 0) {
+            $randomness = (int) ($delay * $this->jitter);
+            $delay += rand(-$randomness, +$randomness);
+        }
+
+        return min((int) $delay, $this->maxBackoff * 1000);
+    }
+
+    public function getMaxAttempts(): int
```

Comment:
I'm in favor of removing the getters/setters, no reason to expose more than needed.

Reply:
I can remove the getters but setters are important as value validation is implemented in the same.

Reply:
Setters can be `private` if you want to keep the validation logic in them.

Reply:
Done